coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -264,6 +264,42 @@ def test_func_dummy(context, inputs):
         # Cleanup the test
         del _TORCH_OPS_REGISTRY.name_to_func_mapping["test_func_dummy"]
 
+
+@pytest.mark.skipif(not _HAS_TORCH, reason=MSG_TORCH_NOT_FOUND)
+class TestFxNodeSupport:
+    @staticmethod
+    def test_simple_case():
+        class Model(torch.nn.Module):
+            def forward(self, a, x, b):
```

Comment:
Yeah that's what I hesitate...
1. If one day that op gets supported, the negative unit test would break
2. Ideally, we do want to support every PyTorch op

Reply:
1. Until thenm we can still have a negative test, and when the failing op is supported, we can modify the testcase?
2. Or is it possible to mock the op name for the negative test?

If none of those are possible, I am good with the current implementation (would still suggest trying 1. though)

Reply:
Added a negative test case by `mock.patch`
```
        with patch.object(
            TorchOpsRegistry,
            "__contains__",
            side_effect=(lambda op_name: op_name == "add"),
        ):
            ...
```

Reply:
magical patch!!!